Modules/Microsoft365DSC/Examples/Resources/EXOReportSubmissionPolicy/1-ConfigureReportSubmissionPolicy.ps1

```diff
@@ -0,0 +1,36 @@
+<#
+This example is used to test new resources and showcase the usage of new resources being worked on.
+It is not meant to use as a production baseline.
+#>
+
+Configuration Example
+{
+    param(
+        [Parameter(Mandatory = $true)]
+        [PSCredential]
+        $Credscredential
+    )
+    Import-DscResource -ModuleName Microsoft365DSC
+
+    node localhost
+    {
+        EXOReportSubmissionPolicy 'ConfigureReportSubmissionPolicy'
+        {
+            IsSingleInstance                       = 'Yes';
```

Comment:
Please remove the semicolons at the end of each line....that is not necessary in PowerShell......it's not C# 😉